### DISCUSSION TITLE
How to remove warning messages

### DISCUSSION THREAD

**Discussion by luizfacury** [neoclide/coc.nvim#3974] [Q&A] [answered]
## Reproduce the bug

**We will close your issue when you don't provide minimal vimrc and we can't
reproduce it**

- Create file `mini.vim` with：

  ```vim
  set nocompatible
  set runtimepath^=/path/to/coc.nvim
  filetype plugin indent on
  syntax on
  set hidden
  ```

- Start (neo)vim with command: `vim -u mini.vim`

- Operate vim.

**Comment by luizfacury** [comment]
Hello, I can`t figure out how to remove the dialogue boxes, it pops up with the following message every time i inicialize neovim:
coc nvim switched to custom popup menu from 0.0.82 you have to change key-mappings for tab to make them work. See :h coc-completion-example

I would like to know how i can disable that or solve the warning.

**Comment by chemzqm** [comment, marked as answer]
You have to tweak key-mappings if you want to use completion of coc.nvim or use `"suggest.autoTrigger": "none"` in coc-settings.json to disable completion of coc.nvim.

**Reply by leofleeo** [reply]
What are those key mappings?

**Reply by fannheyward** [reply]
@CloudyWhale check `:h coc-completion-example` or README.

**Reply by leofleeo** [reply]
that does not tell me what to change

**Reply by chemzqm** [reply]
Change the completion related key-mappings that not work for you

**Reply by leofleeo** [reply]
What were the key mappings changed?

**Reply by chemzqm** [reply]
All of completion related, see `:h coc-completion`

**Reply by feliperafaelbarbosa** [reply]
Thanks, this worked for me.

**Comment by chemzqm** [comment]
Normally you have to replace `pumvisible()` with `coc#pum#visible()` and `"\<C-n>"` with `coc#pum#next(1)`
